docs/install_raspbian.md

```diff
@@ -1,154 +1,217 @@
+`ovos`
 
-First you need to make sure your system is up to date.  It should be close as you just installed a new image.
+As a final configuration step, make sure your system is up to date.
 
 `sudo apt -y update && sudo apt -y upgrade`
 
 We should be done with the basic setup now.  You should have a running RaspberryPiOS device with the user `ovos`
 
-## Step 3: Install OVOS-CORE
+### Step 3: Install ovos-core
 
-There are some recommendations to use a venv for ovos. This guide DOES NOT do that.  The ovos headless stack on a RPi3 is about all it can handle.  It is assumed that this is a dedicated ovos device, therefore no venv is required.
+There are some recommendations to use a venv for OVOS. This guide DOES NOT do that.  The OVOS headless stack on a RPi3 is about all it can handle.  It is assumed that this is a dedicated OVOS device, therefore no venv is required.
 
-There are a few packages required for ovos, so we will install those first
+We will be cloning code from a git repository, so before starting we need to install git.
 
-`sudo apt install build-essential python3-dev python3-pip swig libssl-dev libfann-dev portaudio19-dev libpulse-dev cmake git libncurses-dev pulseaudio-utils`
+`sudo apt install git`
 
-We will be using `pip` to install ovos-core and other related software.  We will also be installing everything to the user environment instead of system wide.  As ovos is the only user, this should be fine.
+We will also be installing everything to the user environment instead of system wide.  As ovos is the only user, this should be fine.
 
-We will assume that everything from here will be done in the home directory of ovos.
+Although not strictly necessary, we assume that we're starting in the ovos home directory.
 
 `cd ~`
 
-Clone this repository
+Clone the repository
 
 `git clone https://github.com/OpenVoiceOS/raspbian-ovos.git`
 
 `cd raspbian-ovos`
 
-Run the install script and follow the prompts
+Run the install script and follow the prompts. It's fine to say yes "Y" to everything.
+
+`./manual_user_install.sh`
+
+<strong>You should now have a running OVOS device!!</strong>
 
-`./manual_install.sh`
+Check your installation with
 
-<strong>You should now have a running ovos device!!</strong>
+`systemctl --user status ovos-*`
 
-Check with this
+The full OVOS can take a few minutes to load (especially on a Pi 3), but the processes should all eventually say `active (running)`, except for `ovos.service` which should say `active (exited)`
 
-`systemctl --user status mycroft*`
+You can also track progress by watching /var/log/syslog.  Once things slow down you can try saying "Hey Mycroft". In a few seconds (the first time is slow) you should hear a 'ding' from the system. Then say "What day is it".  After a delay you should hear information about today's date.
```

Comment:
I would rather point to `~/.local/state/mycroft/<logfile.log>`  That is where they default to even without a systemd service file

Reply:
I updated both references to syslog, changed it to refer users to ~/.local/state/mycroft/*.log